recipes/xrandr.lua

```diff
@@ -0,0 +1,146 @@
+--- Separating Multiple Monitor functions as a separeted module (taken from awesome wiki)
```

Comment:
Is this taken from here: https://awesomewm.org/awesome-www-backup-old/wiki/XRandR_Screen_Table ?
If so, we could acknowledge the author(s) a bit more. I don't know the license conditions of the wiki (technically I bet we are breaking some copyright law), but at least mentioning the authors would be nice.